packages/documentation/docs/controls/forms/_forms-behavior_styleguide.md

```diff
@@ -0,0 +1,42 @@
+Forms behavior refers to the way in which user input is handled and validated within a form. It plays a crucial role in providing a seamless and user-friendly experience for form interactions.
+
+## Validation Feedback
+When data is validated, a validation result is provided to guide users in correcting invalid data, convey a sense of security, warn about critical inputs, or inform about special features of an input.
+
+We support 4 types of validation feedback.
+
+##### Valid
+- native HTML status
```

Comment:
What do you mean with native html status?

Reply:
That is a validation state that is available in HTML natively - compared to info+warning. Does this make any difference for our users e.g. different usage, accessibility etc.?